immersion/kubeflow_pipelines/pipelines/solutions/lab-02_vertex.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>typo: should be "pre-built", not "pre-build"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>